deploy/helm/sumologic/values.yaml

```diff
@@ -1560,6 +1560,9 @@ prometheus-operator:
 falco:
   ## Set the enabled flag to true to enable falco.
   enabled: false
+  image:
+    registry: ghcr.io
+    repository: sumologic/falco
```

Comment:
Isn't `image.tag` missing?

Reply:
The `tag` is used separately, see https://github.com/falcosecurity/charts/blob/master/falco/values.yaml#L6

Reply:
Right, no need to override `tag` because we do not modify it.